cmd/bootstrap/cmd/finalize.go

```diff
@@ -87,11 +90,11 @@ func addFinalizeCmdFlags() {
 	_ = finalizeCmd.MarkFlagRequired("root-commit")
 	_ = finalizeCmd.MarkFlagRequired("epoch-counter")
 
+	finalizeCmd.Flags().BytesHexVar(&flagBootstrapRandomSeed, "random-seed", generateRandomSeed(), "The seed used to for DKG, Clustering and Cluster QC generation")
```

Comment:
what can the seed be? can it be any arbitrary string e.g. "xyz"?

Reply:
Yes, I think it would allow any arbitrary string, but the execution will fail. Using the string `asdssdsdsds` it produces this error `fast KeyGen failed: seeding relic failed: seed length needs to be larger than 16`. Internally I think the cobra library  `hex.DecodeString()` the input, I will add a check to verify that the length of the input string is what the `generateRandomSeed` expects which is a `[48]byte`